### PR TITLE
Use floatThead version 1.2.12

### PR DESCRIPTION
**Pull request by DeeLindesay**
Note to rebuild with new version I had to change the path in build.js as the old one is no longer valid.

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ var postfix = '\n' + [
   '};'
 ].join('\n');
 
-request('https://raw2.github.com/mkoryak/floatThead/v' + process.argv[2] + '/dist/jquery.floatThead.js',
+request('https://cdnjs.cloudflare.com/ajax/libs/floatthead/' + process.argv[2] + '/jquery.floatThead.js',
```
Comment (ForbesLindesay) [inline build.js]: We should still use the version hosted by GitHub as the canonical source.  They have changed their urls recently though, so it should be:

``` js
'https://raw.githubusercontent.com/mkoryak/floatThead/v' + process.argv[2] + '/dist/jquery.floatThead.js'
```

**Comment by ForbesLindesay** [comment]
Once you've reviewed my comment and re-run build, you can merge this.  I'll give you contributor access.

**Comment by DeeLindesay** [comment]
OK Thx

**Comment by DeeLindesay** [comment]
I have changed as requested and merged, but.....
floatThead is now on version 1.2.13 Using the path
`'https://raw.githubusercontent.com/mkoryak/floatThead/v' + process.argv[2] + '/dist/jquery.floatThead.js'` currently fails on v1.2.13.
 Is it always one behind?

**Pull request merged**